packages/react/src/components/Toggletip/index.tsx

```diff
@@ -331,19 +331,21 @@ interface ToggletipContentProps {
  * `children` passed in as a prop inside of `PopoverContent` so that they will
  * be rendered inside of the popover for this component.
  */
-export function ToggletipContent({
-  children,
-  className: customClassName,
-}: ToggletipContentProps) {
+export const ToggletipContent = React.forwardRef<
```

Comment:
I think we might be able to get `displayName` to be what we need/expect by applying it to the forwardRef directly.

```suggestion
const ToggletipContent = React.forwardRef<
```

This way the displayName can be explicitly set before the export lower in the file after the component proptypes

```
ToggletipContent.displayName = "ToggletipContent";
export { ToggletipContent }
```

Then the conditions can in theory be reverted to what they originally were, checking the `displayName`

Reply:
oh, that's better idea, Thanks for the suggestions Taylor, I will do the changes.